src/classiceditorui.js

```diff
@@ -100,10 +98,7 @@ export default class ClassicEditorUI extends BoxedEditorUI {
 		const editor = this.editor;
 
 		const editable = editor.editing.view.getRoot();
-		const editableUI = new EditableUI( editor, editable );
-		const editableUIView = new InlineEditableUIView( editableUI.viewModel, editor.locale );
-
-		editableUI.view = editableUIView;
+		const editableUI = new EditableUI( editor, editable, new InlineEditableUIView( editor.locale ) );
```

Comment:
I'd love if we passed the same params to `EditableUI` that we pass to `StickyToolbar`. Is it possible? Why is it different here?